src/ansys/mechanical/core/embedding/imports.py

```diff
@@ -47,4 +51,23 @@ def global_variables(app: "ansys.mechanical.core.App") -> typing.Dict:
     vars["Quantity"] = Quantity
     vars["System"] = System
     vars["Ansys"] = Ansys
+    vars["Transaction"] = Transaction
```

Comment:
We are missing a section in the documentation for Transaction. @dipinknair could you please file an issue to track this?